client/script.js

```diff
@@ -0,0 +1,184 @@
+window.onload = function (){
+  fetch ("http://localhost:3000/task")
+  .then(response => response.json())
+  .then(todos => {
+    todos.map(item =>{
+      addTodo(item);
+    });
+  })
+}
+
+  function addTodo(item) {
+    const todoItem = document.createElement('li');
+    todoItem.setAttribute('id', item.id);
+    todoItem.innerHTML = item.title;
+    
+    const buttonsContainer = document.createElement('span');
+	
+    const completeButton = document.createElement('button');
+    completeButton.className = 'completeButton';
+    completeButton.setAttribute('taskid', item.id);
+    completeButton.innerHTML = '&#10003';
+
+    const modifyButton = document.createElement('button');
+    modifyButton.className = 'modifyButton';
+    modifyButton.setAttribute('taskid', item.id);
+    modifyButton.innerHTML = '&#9998';
+
+    const deleteButton = document.createElement('button');
+    deleteButton.className = 'deleteButton';
+    deleteButton.setAttribute('taskid', item.id);
+    deleteButton.innerHTML = '&#128465';
+
+    buttonsContainer.append(completeButton, modifyButton, deleteButton);
+    todoItem.appendChild(buttonsContainer);
+
+    if(item.done){
+      todoItem.classList.add('completed');
+    }
+
+    document.getElementById('todoList').appendChild(todoItem);
+
+    const completeButtons = todoItem.querySelectorAll('button.completeButton');
+    completeButtons.forEach(button => {

```

Comment:
hi 
i guess you've only one complete button, declared a few lines above as 
```js
 const completeButton = document.createElement('button');
```
please use that one here instead of 
```js
const completeButtons = todoItem.querySelectorAll('button.completeButton');
```